exonum-java-binding/cryptocurrency-demo/src/test/java/com/exonum/binding/cryptocurrency/transactions/CreateWalletTxTest.java

```diff
@@ -69,60 +67,51 @@ void constructorRejectsNegativeBalance() {
     Throwable t = assertThrows(IllegalArgumentException.class,
         () -> new CreateWalletTx(initialBalance));
 
-    assertThat(t.getMessage(), equalTo("The initial balance (-1) must not be negative."));
+    assertThat(t.getMessage()).isEqualTo("The initial balance (-1) must not be negative.");
   }
 
   @Test
   @RequiresNativeLibrary
-  void executeCreateWalletTx() throws Exception {
-    CreateWalletTx tx = new CreateWalletTx(DEFAULT_INITIAL_BALANCE);
-
-    try (Database db = MemoryDb.newInstance();
-        Cleaner cleaner = new Cleaner()) {
-      Fork view = db.createFork(cleaner);
-
-      // Execute the transaction
-      TransactionContext context = newContextBuilder(view)
-          .authorPk(OWNER_KEY)
-          .build();
-      tx.execute(context);
-
-      // Check that entries have been added.
-      CryptocurrencySchema schema = new CryptocurrencySchema(view);
-      MapIndex<PublicKey, Wallet> wallets = schema.wallets();
+  void executeCreateWalletTx() {
+    try (TestKit testKit = TestKit.forService(CryptocurrencyServiceModule.class)) {
+      KeyPair emulatedNodeKeyPair = testKit.getEmulatedNode().getServiceKeyPair();
```

Comment:
Is there a reason to use the emulated node key instead of a predefined key? It is unclear why it is
needed (if it is).